docs/controller.md

```diff
@@ -49,7 +49,10 @@
     - **value**: `str`
 
 - **emulator-read-and-confirm-mnemonic**
-  - **action**: simulates the backup process
+  - **action**: simulates the classic backup process
```

Comment:
Not that it matters much but we usually call it Single backup. Or technically BIP-39.